content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -62,7 +62,7 @@ have some advantages for start-up related code:
 * Init containers can contain utilities or custom code for setup that are not present in an app
   image. For example, there is no need to make an image `FROM` another image just to use a tool like
   `sed`, `awk`, `python`, or `dig` during setup.
-* Init containers can securely run utilities that would make an app container image less secure.
+* Init containers can securely run utilities that would make an app container image more secure.
```

Comment:
The point being made here is that the utility is somehow dangerous and including it in the app container image adds risk.
For example, “distroless” containers are more difficult to attack than containers from images that include a handy set of tools.

@nayaksuraj adding utilities to the app container _adds_ risk (or, at least, might add risk). Using an init container lets things happen whilst still keeping the app container's attack surface small.
I hope that makes sense and helps guide you towards tweaking this pull request to help future readers.

Reply:
@nayaksuraj do you have any more questions?

Reply:
"Init containers can securely run utilities that would otherwise make an app container image less secure."?

Reply:
See https://github.com/kubernetes/website/issues/16686 for more discussion on this as well to hopefully make it more clear. This seems like a common point of misunderstanding so re-wording it would definitely help!

Reply:
@sftim No questions, it's quite clear to me. you explained very well.